packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -581,8 +584,13 @@ mixin _CupertinoSheetRouteTransitionMixin<T> on PageRoute<T> {
     );
   }
 
+  /// Determines whether the content can be scrolled.
```

Comment:
Nit: replace "scroll" with "drag". So "can be dragged to dismiss the sheet", "dragging is enabled". Scrolling refers to a slightly different interaction.

Reply:
Also, it would be better if this was moved above the `buildPage` method https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#order-other-class-members-in-a-way-that-makes-sense

Reply:
fix this commit
https://github.com/flutter/flutter/pull/163923/commits/0ebeb398b34fcd6ff7c7c1e3940236628e15fbd5